dev-guide/development.md

```diff
@@ -8,24 +8,47 @@ Before you begin, check the [supported platforms](./requirements.md#supported-pl
 
 ## Build TiKV
 
-After you install the RocksDB shared library, you can build TiKV directly without `ROCKSDB_SYS_STATIC`.
+### Rust toolchain
```

Comment:
This is conflicted with https://github.com/MyonKeminta/docs/blob/1693df64c7d0278e26416bc271423000292e1a09/dev-guide/requirements.md#prerequisites, I think you should move these to the prerequisites.